BUILD.md

```diff
@@ -93,4 +93,4 @@ The Markdown renderer is located under ReactNativeClient/lib/joplin-renderer. Wh
 
 # Troubleshooting
 
-Please read for the [Build Troubleshooting Document](https://github.com/laurent22/joplin/blob/master/readme/build_troubleshooting.md) for various tips on how to get the build working.
\ No newline at end of file
+Please read for the [Build Troubleshooting Document](https://github.com/laurent22/joplin/blob/master/readme/build_troubleshooting.md) for various tips on how to get the build working.
```

Comment:
Thank you for the pull request, it makes sense. Please could you just remove this change from the pull request as it's unrelated.

Reply:
Apologize for that. I guess that it was due to a merge conflict between the current codebase and the ones I have for PR #1922. I am trying to fix this.